src/test/datascience/ipywidgets/samples/beakerx/LICENSE

```diff
@@ -0,0 +1,201 @@
+Apache License
```

Comment:
I wanted to use the actual extension.js files for tests instead of crafting our own content.
Included the license files along with the extesnsion.js files.

Reply:
Wondering if some automated check tool is going to flag us for having a different license in the repo. But I guess we'll see if anything like that crops up moving forward.